.github/workflows/nightly.yaml

```
@@ -0,0 +1,11 @@
+# Schedule to trigger Main workflow with cleared cache
+name: nightly
+on:
+  schedule:
+    - cron: '0 2 * * *'
```

```suggestion
    - cron: '0 5 * * *'
```
Maybe a bit later? 2am UTC is 6pm pacific in winter, which could be annoying if someone works late. We don't want to add more jobs than necessary to our limited concurrency pool during work hours

---

agreed